File: src/io/innofang/proxy/example/RealPicker.java
package io.innofang.proxy.example;

/**
 * Created by Inno Fang on 2017/4/8.
 * Real Picker 需要取件的人 （真实取件人）
 */
public class RealPicker implements IPicker {
    @Override
    public void receiveMessage() {
        System.out.println("Receive text message");
    }

    @Override
    public void takeCourier() {
        System.out.println("Take the courier");
    }

    @Override
    public void signatureAcceptance() {
        System.out.println("Signature Acceptance");
    }
}

File: src/io/innofang/proxy/example/kotlin/RealPicker.kt
package io.innofang.proxy.example.kotlin

/**
 * Created by Inno Fang on 2017/9/3.
 */
class RealPicker: IPicker {
    override fun receiveMessage() {
        println("Receive text message")
    }

    override fun takeCourier() {
        println("Take the courier")
    }

    override fun signatureAcceptance() {
        println("Signature acceptance")
    }
}